test/wasm2js/atomic_fence.2asm.js

```diff
@@ -24,9 +24,8 @@ function asmFunc(global, env, buffer) {
   
  }
  
- var FUNCTION_TABLE = [];
  function __wasm_memory_size() {
-  return buffer.byteLength / 65536 | 0;
+  return buffer.byteLength >> 16 | 0;
```

Comment:
Recently v8 could handle `4GB`, so perhaps better use unsigned right shift? wdyt?

Reply:
I think you mean unsigned, not signed? But yes, for 4GB we do need `>>>` here, good point.